opensrp-chw/src/main/java/org/smartregister/chw/dao/ChwChildDao.java

```diff
@@ -184,4 +184,16 @@ public Void readCursor(Cursor cursor) {
         AbstractDao.readData(sql, dataMap);
         return childExists[0] && motherAlive[0];
     }
+
+    public static List<String> getRegisteredCertificateNumbers() {
```

Comment:
What would  be the performance impact when using a DHO account with 1K+ children?